main.js

```diff
@@ -1,36 +1,25 @@
 import Expo from 'expo';
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
-import { Provider} from 'react-redux';
+import { Provider } from 'react-redux';
 import { createStore, applyMiddleware } from 'redux';
 import logger from 'redux-logger';
 
```

Comment:
I sometimes do this to logically separate vendor vs. app code imports.  Removed.